<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>collector</class>
 <widget class="QMainWindow" name="collector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>collector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoButt">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>解析视频流</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButt">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重定位视频文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>79</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;路径&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>从</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButt">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endButt">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>190</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成当前数据到txt文件</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>230</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="lastButt">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上一帧A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButt">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>下一帧D</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;文件名&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="btLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>470</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;拇指尖坐标&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="tfjLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;掌心坐标&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bjLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>490</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;拇指关节坐标&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="tlLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;左上角坐标&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="brLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;右下角坐标&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="iftLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>410</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;食指指尖坐标&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="ifjLabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;食指关节坐标&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dropButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>260</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>劣质图片S</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
